docs/RELEASES.md

```diff
@@ -1,11 +1,12 @@
 # CloudEvents Releases
 
-| Specification Group | Version | Release Date | Release Notes |
-| :------------------ | :-----: | :----------- | :-----------: |
-| CloudEvents         | [1.0.2](https://github.com/cloudevents/spec/tree/v1.0.2/cloudevents) | 2022/02/05 | [Notes](../cloudevents/RELEASE_NOTES.md#v102---20220205) |
-| CloudEvents         | [1.0.1](https://github.com/cloudevents/spec/tree/v1.0.1) | 2020/12/12 | [Notes](../cloudevents/RELEASE_NOTES.md#v101---20201212) |
-| CloudEvents         | [1.0](https://github.com/cloudevents/spec/tree/v1.0) | 2019/10/24 | [Notes](../cloudevents/RELEASE_NOTES.md#v100---20191024) |
-| CloudEvents         | [0.3](https://github.com/cloudevents/spec/tree/v0.3) | 2019/06/13 | [Notes](../cloudevents/RELEASE_NOTES.md#v03---20190613) |
-| CloudEvents         | [0.2](https://github.com/cloudevents/spec/tree/v0.2) | 2018/12/06 | [Notes](../cloudevents/RELEASE_NOTES.md#v02---20181206) |
-| CloudEvents         | [0.1](https://github.com/cloudevents/spec/tree/v0.1) | 2018/04/20 | [Notes](../cloudevents/RELEASE_NOTES.md#v01---20180420) |
+| Specification Group | Version                                                              | Release Date | Release Notes                                            |
+| :------------------ | :------------------------------------------------------------------: | :----------- | :------------------------------------------------------: |
+| CESQL               | 1.0                                                                  | 2024/06/13   | [Notes](../cesql/RELEASE_NOTES.md#v100---20240613)       |
```

Comment:
let's make `1.0` a link to: https://github.com/cloudevents/spec/tree/cesql/v1.0.0
the link checker will fail until we create the new release/branch but that's ok